### PR TITLE
Add partner callback links to map infowindows

### PR DESCRIPTION
**Pull request by smattingly**
This can be tested from https://dewv.net/ftm/test.html which I temporarily updated to embed the heroku deployment.

### CODE REVIEW

File: client/locations-list-map.js [hunk 1]
```diff
@@ -382,8 +382,15 @@ function createRequesterMarkerContent(entry, separator) {
...
+  const header = separator ? ce('h5', 'separator', ctn(name)) : ce('h5', null, ctn(name));
+  const headerPartnerLink = createPartnerLinkIcon(entry.row_id);
+
+  if (headerPartnerLink) {
+    ac(header, headerPartnerLink);
+  }
+
   // Text to go into InfoWindow
-  const contentTags = separator ? [ce('h5', 'separator', ctn(name))] : [ce('h5', null, ctn(name))];
+  const contentTags = [header];
```
Comment (ngottlieb) [inline client/locations-list-map.js:393]: we have a bit of overlap in this area, but it'll be easy to resolve and I can do it on my branch.

Comment (smattingly) [reply client/locations-list-map.js:393]: The links are the "D&E" images that match the site favicon.

Comment (ngottlieb) [reply client/locations-list-map.js:393]: Yeah, it's not that I don't see the links, it's that they're not there. And the `body` tag doesn't have the partner data attributes set...

Comment (smattingly) [reply client/locations-list-map.js:393]: Do you mean that you don't see the red-circled icon/link at the test URL shown?

image

Comment (smattingly) [reply client/locations-list-map.js:393]: Yes, I think that is what you mean. I went to a different machine and don't see them either.

Comment (ngottlieb) [reply client/locations-list-map.js:393]: Yes, I mean exactly that.

I also mean (when I investigated the issue) that the variables your code is setting in the router aren't being set in my browser. I'm using Brave, it's a Chromium based browser.

Comment (ngottlieb) [reply client/locations-list-map.js:393]: I also don't see them in Firefox. They're there in Safari.

Comment (ngottlieb) [reply client/locations-list-map.js:393]: Hmm, this must be some kind of caching issue, but I can't get it to resolve in Brave...

I just opened Chrome (a browser I don't use on this computer) and it worked. So I guess don't worry about it, some sort of local quirk.

**Comment by ngottlieb** [comment]
@smattingly I *think* this is a real issue in Brave, but it may be unique to Brave? I asked @nburt if he can replicate since I think he uses Brave.

**Comment by ngottlieb** [comment]
So @nburt is seeing the same thing in Brave.

I'm looking into the browser spec here and I'm not sure that we can use referrer safely. Behavior seems to be inconsistent, particularly with HTTPS, and it also seems like browsers are not supposed to transmit the original website's URL as a referrer header within an iframe in https though some obviously do.

Maybe we can try to tweak this by using the `referrerpolicy` attribute? https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referrer-Policy

**Comment by smattingly** [comment]
@ngottlieb No, it seems the only place it works are the browser*s* on my dev machine.

**Comment by ngottlieb** [comment]
@smattingly not quite, it works on some of my browsers. I think it depends on the browser implementation of `referrer` headers within iframes.

**Comment by smattingly** [comment]
The referrer policy didn't seem to be help, so I went back to the earlier idea of having the link partner identify themselves with a query parameter.

**Pull request merged**